src/Components/Auth/CustomRegisterForm.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import '../FormStyles.css';
+import { useFormik } from 'formik';
+
+const CustomRegisterForm = () => {
+    const [initialValues, setInitialValues] = useState({
+        email: '',
+        password: ''
+    });
+
+    const validate = values => {
```

Comment:
this function should go on the validations file in the Utils folder. But it's not blocking for this first PR